spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -82,7 +82,9 @@ object SpdxLicenseAliasMapping {
         "GPL2" to GPL_2_0_ONLY,
         "gpl3" to GPL_3_0_ONLY,
         "GPLv2" to GPL_2_0_ONLY,
+        "GPLv2+" to GPL_2_0_OR_LATER,
         "GPLv3" to GPL_3_0_ONLY,
+        "GPLv3+" to GPL_2_0_OR_LATER,
```

Comment:
But this should map to `GPL_3_0_OR_LATER`.

Reply:
Fixed it